centos-7.template

```diff
@@ -166,6 +168,18 @@ EOF
 
 systemctl enable docker
 
+# Show a warning banner when using yum to install software
+mv /usr/bin/yum /usr/bin/yum-unsupported
+# Place holder for base64 encode yum-wrapper script
+cat > yum-wrapper.base64 << EOF
```

Comment:
I think we could simplify this inclusion of encoded data even more, almost to a degree where you just do something like '${yum_wrapper}'.  The current approach is quite fragile.